examples/basic/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "private": "true",
+  "scripts": {
+    "start": "react-scripts start"
+  },
+  "dependencies": {
+    "@razorpay/blade": "../../packages/blade/build/",
```

Comment:
Codesandbox CI overrides this with their own preview version, Storybook overrides it with `*`